.github/ISSUE_TEMPLATE/enhancement.yaml

```diff
@@ -7,6 +7,19 @@ body:
     attributes:
       value: |
         Thanks for taking the time to fill out this proposal / feature reqeust
+  - type: dropdown
+    id: urgency
+    attributes:
+      label: Urgency
+      description: How urgent is this proposal?
+      options:
+      - I don't know
+      - Low (it can wait)
+      - Medium (it should be backlogged)
+      - High (it should be prioritized)
+      - Critical (it should be worked on immediately)
+    validations:
+      required: true
```

Comment:
Same here, i think its up on us to define how urgent something is and not on the community/customer